### PR TITLE
Added help text for physics shards

### PR DESCRIPTION
**Pull request by ShahFragcolor**

### CODE REVIEW

File: shards/modules/anim/anim.cpp [hunk 1]
```diff
@@ -173,17 +180,17 @@ struct TimerShard {
 };
 
 struct PlayShard {
-  static SHOptionalString help() { return SHCCSTR(R"(Outputs the duration of an animation, in seconds)"); }
+  static SHOptionalString help() { return SHCCSTR("Plays the animation of the glTF model specified in the Animation parameter."); }
```
Comment (guusw) [inline shards/modules/anim/anim.cpp]: I'd say something like "evaluates the animation at the given time".
Since it technically doesn't "Play" the animation, just interpolates all the timelines in the animation and gives you the output, which you then have to apply to the mesh to actually "play"

File: shards/modules/physics/body.cpp [hunk 2]
```diff
@@ -35,44 +35,53 @@ static ParamTypeMask &operator|=(ParamTypeMask &lhs, ParamTypeMask rhs) {
...
+  }
 
   static inline Type PhysicsDOFSeqType = Type::SeqOf(PhysicsDOFEnumInfo::Type);
   static inline Types PhysicsDOFTypes{PhysicsDOFEnumInfo::Type, PhysicsDOFSeqType, Type::VariableOf(PhysicsDOFSeqType)};
 
-  PARAM_PARAMVAR(_location, "Location", "The initial location, updated by physics simulation", {shards::CoreInfo::Float3VarType});
-  PARAM_PARAMVAR(_rotation, "Rotation", "The initial location, updated by physics simulation", {shards::CoreInfo::Float4VarType});
-  PARAM_VAR(_static, "Static", "Static node, persist when not activated", {shards::CoreInfo::BoolType});
-  PARAM_PARAMVAR(_enabled, "Enabled", "Can be used to toggle this node when it has static persistence",
+  PARAM_PARAMVAR(_location, "Location", "The initial location of the physics object. The variable provided in this parameter is also updated through the physics simulations conducted on this body.", {shards::CoreInfo::Float3VarType});
```
Comment (guusw) [inline shards/modules/physics/body.cpp]: You can update these, since this value will now always be applied to the body when changed

Comment (guusw) [reply shards/modules/physics/body.cpp]: Same for rotation

File: shards/modules/physics/body.cpp [hunk 3]
```diff
@@ -35,44 +35,53 @@ static ParamTypeMask &operator|=(ParamTypeMask &lhs, ParamTypeMask rhs) {
...
+  PARAM_PARAMVAR(_rotation, "Rotation", "The initial rotation of the physics object. The variable provided in this parameter is also updated through the physics simulations conducted on this body.", {shards::CoreInfo::Float4VarType});
+  PARAM_VAR(_static, "Static", "If false, the physics body will be destroyed when the shard is not called. If true, the body will persist and be included in the physics simulation even if the shard is not called.", {shards::CoreInfo::BoolType});
```
Comment (guusw) [inline shards/modules/physics/body.cpp]: Not sure if we used the terminology "call" with respect to shards before, "activate" is the correct one but not sure which we use in the docs

File: shards/modules/physics/constraints.cpp [hunk 4]
```diff
@@ -343,31 +341,31 @@ struct DistanceConstraint final : public ConstraintBase {
...
-  PARAM_PARAMVAR(_limitSpring, "LimitSpring", "When set, makes the limits soft", SpringSettings::TypesVarOrNone);
+  PARAM_PARAMVAR(_limitSpring, "LimitSpring", "When set, makes the limits soft.", SpringSettings::TypesVarOrNone);
 
   PARAM_PARAMVAR(_maxFrictionForce, "MaxFrictionForce",
-                 "Maximum amount of friction force to apply (N) when not driven by a motor",
+                 "Maximum amount of friction force to apply (in Newtons) on the slider.",
                  {CoreInfo::FloatType, CoreInfo::FloatVarType});
 
-  PARAM_PARAMVAR(_motorSettings, "MotorSettings", "Motor settings around the sliding axis", {MotorSettings::TypesVarOrNone});
+  PARAM_PARAMVAR(_motorSettings, "MotorSettings", "Defines the maximum and minimum amount of force applied to the slider as it slides along the axis.", {MotorSettings::TypesVarOrNone});
```
Comment (guusw) [inline shards/modules/physics/constraints.cpp]: This is meant for motors, where the body is driven by a "motor" along the slider. Although I am still missing the part that actually enables this, so it's for future use.

File: shards/modules/physics/constraints.cpp [hunk 5]
```diff
@@ -386,7 +384,7 @@ struct SliderConstraint final : public ConstraintBase {
     _limitsMax = Var(std::numeric_limits<float>::max());
     _limitSpring = SpringSettings::Default;
     _maxFrictionForce = Var(0.0f);
-    _motorSettings = MotorSettings::Default;
+    _motorSettings = SpringSettings::Default;
```
Comment (guusw) [inline shards/modules/physics/constraints.cpp]: Undo this change

File: shards/modules/physics/physics.cpp [hunk 6]
```diff
@@ -450,7 +479,11 @@ struct MotionTypeShard {
 struct SetPoses {
   static SHTypesInfo inputTypes() { return SHBody::Type; }
   static SHTypesInfo outputTypes() { return SHBody::Type; }
-  static SHOptionalString help() { return SHCCSTR("Override poses of the physics body"); }
+  static SHOptionalString help() { return SHCCSTR("Overwrites the position and rotation of the input physics body"); }
+  static SHOptionalString inputHelp() { return SHCCSTR("The physics body to overwrite the position and rotation of."); }
```
Comment (guusw) [inline shards/modules/physics/physics.cpp]: Worth noting that setting only position OR rotation is possible too

File: shards/modules/physics/physics.cpp [hunk 7]
```diff
@@ -486,7 +519,11 @@ struct SetPoses {
 struct SetVelocities {
   static SHTypesInfo inputTypes() { return SHBody::Type; }
   static SHTypesInfo outputTypes() { return SHBody::Type; }
-  static SHOptionalString help() { return SHCCSTR("Override velocity of the physics body"); }
+  static SHOptionalString help() { return SHCCSTR("Overwrites the linear and angular velocity of the input physics body"); }
+  static SHOptionalString inputHelp() { return SHCCSTR("The physics body to overwrite the linear and angular velocity of."); }
```
Comment (guusw) [inline shards/modules/physics/physics.cpp]: Worth noting that setting only linear OR angular is possible too

File: shards/modules/physics/physics.cpp [hunk 8]
```diff
@@ -524,7 +561,20 @@ template <int Mode> struct ApplyVelocity {
 
   static SHTypesInfo inputTypes() { return SHBody::Type; }
   static SHTypesInfo outputTypes() { return SHBody::Type; }
-  static SHOptionalString help() { return SHCCSTR("Applies a force to the physics body"); }
+
+  static SHOptionalString help() {
+    if constexpr (Mode == 0) {
+      return SHCCSTR("This shard applies a continuous force linear or angular force to the input physics body.");
```
Comment (guusw) [inline shards/modules/physics/physics.cpp]: use "and/or"

**Comment by sinkingsugar** [comment]
We done here yet? Ready to merge?
